examples/Advanced-Building-your-own-architecture.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<p>Is it <code>Feature Blocks</code>  or <code>Features Blocks</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I used the same naming as in the repo folder (which is <code>Features</code>, and I think it's used to showcase the support of different types of features). Does <code>Feature Blocks</code> sound better for you? </p>